src/posts/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layouts/post
+title: News | floe
+permalink: posts/index.html
+---
+<div id="content" class="floe-content">
+    <div class="flc-toc-tocContainer toc"> </div>
+    <h2> News </h2>
+    <div class="floe-news-archive">
+        <ul id="reverseneed">
+            {%- for post in collections.post -%}
+              <li id="reverseneed"><a href="{{ '/' | url }}{{ post.data.permalink }}"><p> {{ post.data.title }}</p></a> 
```

Comment:
If using keyboard Tab key to navigate the post index, focus moves from the bottom of the page toward the top. This is a problem caused by the CSS transform rotation but the DOM order remaining the same. Ideally the DOM order should be consistent with Tab focus order so that people using only keyboards or switches (or other forms of navigation) don't get confused by focus moving backward. 

To fix this, I can see 2 ways:
1. When creating the Post collection, ensure the contents are sorted in reverse date order, or
2. In the loop traverse the collection in reverse so that the last collection item is first in the `ul` list.

Option 1 is the preferred method as it keeps the logic for data handling in the controller, leaving the template to just render the contents. Option 2 is a bit of a hack and should only be used if Option 1 isn't possible for some reason.

Side comment: `id` values should be unique in a document. The way `id` is being used here should really be a `class` instead. But if we use Option 1 or 2 above, we can get rid of these `id`s and remove the rotation CSS as it won't be needed anymore.

Reply:
fixed